src/mod/applications/mod_conference/conference_loop.c

```diff
@@ -1569,10 +1569,10 @@ void conference_loop_output(conference_member_t *member)
 		if (conference_utils_member_test_flag(member, MFLAG_INDICATE_MUTE)) {
 			if (!zstr(member->conference->muted_sound)) {
 				conference_member_play_file(member, member->conference->muted_sound, 0, SWITCH_TRUE);
-			} else {
+			} else if (!zstr(member->conference->muted_tts)) {
 				char msg[512];
 
-				switch_snprintf(msg, sizeof(msg), "Muted");
+				switch_snprintf(msg, sizeof(msg), member->conference->muted_tts);
 				conference_member_say(member, msg, 0);
 			}
```

Comment:
Don't we need a default tts sound like before?
```
else {
	char msg[512];
	switch_snprintf(msg, sizeof(msg), "Muted");
	conference_member_say(member, msg, 0);
}
```

Reply:
No, they are needed to be configured in the profile instead if we want the profile to play TTS muted sound